R/utils.R

```diff
@@ -119,26 +119,15 @@ add_facet_labels <- function(p, xfacet_label = NULL, yfacet_label = NULL) {
   })
 }
 
-#' Call a function with a character vector for the \code{...} argument
+#' Call a function with a character vector for the `...` argument
 #'
-#' @param fun (`character`) Name of a function where the \code{...} argument
-#'   shall be replaced by values from \code{str_args}.
-#' @param str_args (`character`) A character vector that the function shall
-#'  be executed with
+#' @param fun (`character`) Name of a function where the `...` argument shall be replaced by values from `str_args`.
+#' @param str_args (`character`) A character vector that the function shall be executed with
```

Comment:
```suggestion
#' @param str_args (`character`) A character vector that the function shall be executed with.
```

Reply:
> (`character`) A character

: P ? so many characters

Reply:
Because way back when sb wanted the type to not be part of the sentence. Am I the only one who remembers things like that?

Reply:
Just saying. I actually remember that there's gonna be a separate PR just to cover those cases. This was here to bring some joy and smile : D resolving.

Reply:
I'll pass the joy to the author of that PR then :P